odoo/addons/base/views/res_partner_views.xml

```diff
@@ -668,7 +668,7 @@
             <field name="name">Other Partners</field>
             <field name="type">ir.actions.act_window</field>
             <field name="res_model">res.partner</field>
-            <field name="domain">[('supplier','!=',1),('customer','!=',1)]</field>
+            <field name="domain">[('supplier_rank','=',0),('customer_rank','=',0)]</field>
```

Comment:
but this fields are defined in `account`, not in `base` 🤔

Reply:
shit :p